autobotAI_integrations/integrations/palo_alto_scm/__init__.py

```
@@ -0,0 +1,237 @@
+                "grant_type": "client_credentials",
+                "client_id": self.integration.client_id,
+                "client_secret": self.integration.client_secret,
+                "scope": self.integration.scope,
+            },
+            timeout=30,
+        )
+        if not response.ok:
+            raise requests.exceptions.HTTPError(
+                f"{response.status_code} - {response.text}",
+                response=response,
+            )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)